cran-comments.md

```diff
@@ -0,0 +1,17 @@
+# Release summary
+
+This is a resubmission. In this version, I have:
+
+* Fixed the URLs in the DESCRIPTION file.
+* Added a new file ([WORDLIST](inst%2FWORDLIST)) to allow custom words
```

Comment:
Can't we use in the markdown link a non encoded slash `/` character instead? The same comment applies to the link at the end of the page